pkg/agent/agent.go

```diff
@@ -226,6 +229,22 @@ func handleCentralConfig(centralCfg config.CentralConfig) error {
 		resource.MergeComplianceAgentWithConfig(agent.agentResourceManager.GetAgentResource(), centralCfg)
 	}
 
+	// do not get entitlements in test
+	if !util.IsNotTest() {
```

Comment:
Probably just me and my simplistic mind :) ... the ! (not) is messing with me.
maybe for readability, if util.IsTest() {return nil}